pkgs/by-name/to/tomlplusplus/package.nix

```diff
@@ -0,0 +1,67 @@
+{
+  cmake,
+  fetchFromGitHub,
+  fetchpatch2,
+  glibcLocales,
+  lib,
+  meson,
+  ninja,
+  nix-update-script,
+  pkg-config,
+  stdenv,
+  testers,
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "tomlplusplus";
+  version = "3.4.0";
+
+  src = fetchFromGitHub {
+    owner = "marzer";
+    repo = "tomlplusplus";
+    rev = "v${finalAttrs.version}";
+    hash = "sha256-h5tbO0Rv2tZezY58yUbyRVpsfRjY3i+5TPkkxr6La8M=";
+  };
+
+  patches = [
+    # https://github.com/marzer/tomlplusplus/pull/233
+    (fetchpatch2 {
```

Comment:
How about adding a simple comment asking to eventually remove it at the next update?